deploy/backplane/10-srep-admins-cluster.ClusterRole.yml

```
@@ -59,3 +59,48 @@ rules:
   - '*'
   verbs:
   - '*'
+#### Start - more perms for srep
+# https://issues.redhat.com/browse/OSD-5537
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - '*'
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - selfsubjectaccessreviews
+  verbs:
+  - '*'
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - selfsubjectrulesreviews
+  verbs:
+  - '*'
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - '*'
+- apiGroups:
+  - authorization.openshift.io
+  resources:
+  - resourceaccessreviews
+  verbs:
+  - '*'
+- apiGroups:
+  - authorization.openshift.io
+  resources:
+  - subjectaccessreviews
+  verbs:
+  - '*'
```

These can be collapsed:
```suggestion
- apiGroups:
  - authorization.k8s.io
  resources:
  - selfsubjectaccessreviews
  - selfsubjectrulesreviews
  - subjectaccessreviews
  verbs:
  - '*'
- apiGroups:
  - authorization.openshift.io
  resources:
  - resourceaccessreviews
  - subjectaccessreviews
  verbs:
  - '*'
```

---

Thanks @cblecker . I will refactor the new PR : https://github.com/openshift/managed-cluster-config/pull/534 accordingly.